Formula/telegram-cli.rb

```diff
@@ -0,0 +1,60 @@
+class TelegramCli < Formula
+  desc "Command-line interface for Telegram."
+  homepage "https://github.com/vysheng/tg"
+  # Use the tag instead of the tarball to get submodules
+  url "https://github.com/vysheng/tg.git",
+      :tag => "1.3.1",
+      :revision => "5935c97ed05b90015418b5208b7beeca15a6043c"
+  head "https://github.com/vysheng/tg.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "readline"
+  depends_on "libevent"
+  depends_on "openssl"
+  depends_on "libconfig" => :recommended
+  depends_on "jansson" => :recommended
+  depends_on "lua" => :recommended
+  depends_on "python" => :recommended
```

Comment:
> It’s nice to have the option though, isn’t it? I mean, they’re already on by default (they should be —without-* options). Maybe libconfig I can make essential, if you prefer.

We prefer to remove options unless they provide really useful functionality. Removing dependencies isn't one, really.